src/components/DeviceCPULoad.tsx

```diff
@@ -0,0 +1,100 @@
+import StandartLineChart from './StandardLineChart'
+import ShadowBox from './ShadowBox'
+import IconText from './IconText'
+import { Paragraph, Separator, XStack, YStack } from 'tamagui'
+
+const data = [
+  {
+    id: 'cpu',
+    color: '#8DC6BC',
+    data: [
+      {
+        x: '1',
+        y: 15,
+      },
+      {
+        x: '2',
+        y: 12,
+      },
+      {
+        x: '3',
+        y: 43,
+      },
+      {
+        x: '4',
+        y: 20,
+      },
+      {
+        x: '5',
+        y: 60,
+      },
+      {
+        x: '6',
+        y: 5,
+      },
+      {
+        x: '7',
+        y: 15,
+      },
+      {
+        x: '8',
+        y: 12,
+      },
+      {
+        x: '9',
+        y: 43,
+      },
+      {
+        x: '10',
+        y: 20,
+      },
+      {
+        x: '11',
+        y: 60,
+      },
+      {
+        x: '12',
+        y: 132,
+      },
+    ],
+  },
+]
+const DeviceCPULoad = () => {
```

Comment:
Let's take the CPU load entries here as an array of load values and then you can generate what you currently call `data` from it. The idea is for this component to take that array and do the right thing. Some other part of the system makes sure we have that array, and so on.